idaes/models/unit_models/heat_exchanger_1D.py

```diff
@@ -489,132 +515,143 @@ def _make_performance(self):
         if self.config.has_wall_conduction == WallConductionType.zero_dimensional:
             self.temperature_wall = Var(
                 self.flowsheet().time,
-                self.tube.length_domain,
+                self.cold_side.length_domain,
                 initialize=298.15,
-                units=shell_units("temperature"),
+                units=hot_side_units("temperature"),
             )
 
             # Performance equations
-            # Energy transfer between shell and tube wall
+            # Energy transfer between hot_side and cold_side wall
 
             @self.Constraint(
                 self.flowsheet().time,
-                self.shell.length_domain,
-                doc="Heat transfer between shell and tube",
+                self.hot_side.length_domain,
+                doc="Heat transfer between hot_side and cold_side",
             )
-            def shell_heat_transfer_eq(self, t, x):
-                return self.shell.heat[t, x] == -self.N_tubes * (
-                    self.shell_heat_transfer_coefficient[t, x]
+            def hot_side_heat_transfer_eq(self, t, x):
+                return self.hot_side.heat[t, x] == -self.N_tubes * (
+                    self.hot_side_heat_transfer_coefficient[t, x]
                     * c.pi
-                    * self.d_tube_outer
+                    * self.d_cold_side_outer
                     * (
-                        self.shell.properties[t, x].temperature
+                        self.hot_side.properties[t, x].temperature
                         - self.temperature_wall[t, x]
                     )
                 )
 
-            # Energy transfer between tube wall and tube
+            # Energy transfer between cold_side wall and cold_side
             @self.Constraint(
                 self.flowsheet().time,
-                self.tube.length_domain,
+                self.cold_side.length_domain,
                 doc="Convective heat transfer",
             )
-            def tube_heat_transfer_eq(self, t, x):
-                return self.tube.heat[t, x] == self.tube_heat_transfer_coefficient[
+            def cold_side_heat_transfer_eq(self, t, x):
+                return self.cold_side.heat[
+                    t, x
+                ] == self.cold_side_heat_transfer_coefficient[
                     t, x
                 ] * c.pi * pyunits.convert(
-                    self.d_tube_inner, to_units=tube_units("length")
+                    self.d_cold_side_inner, to_units=cold_side_units("length")
                 ) * (
                     pyunits.convert(
-                        self.temperature_wall[t, x], to_units=tube_units("temperature")
+                        self.temperature_wall[t, x],
+                        to_units=cold_side_units("temperature"),
                     )
-                    - self.tube.properties[t, x].temperature
+                    - self.cold_side.properties[t, x].temperature
```

Comment:
I'm confused about how heat is defined in the `ControlVolume1D`. I thought it was heat per unit length for the entire flow. This gives heat transfer out of a single tube, but we would actually want heat transfer out of _every_ tube, right? Am I missing something? Compare the [2d boiler heat exchanger](https://github.com/IDAES/idaes-pse/blob/5657571fde5eaa8f9cf37c29f150be2a78ffa8ce/idaes/models_extra/power_generation/unit_models/boiler_heat_exchanger_2D.py#L1630).